lib/normalize-option.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var valueOrFunction = require('value-or-function');
+
+module.exports = function normalizeOption(types, defaultValue, value) {
```

Comment:
The best would be something like `typed(valueOrFunction(defaultValue(option, defaultValue), 'string', 'object')`. The fact that the `value-or-function` does both "value or function" and "typed" without supporting multiple types is limiting.